### PR TITLE
Fixed wrong option alias for enb-beautify

### PR DESCRIPTION
**Pull request by Realetive**

### CODE REVIEW

File: app/templates/.enb/make.js [hunk 1]
```diff
@@ -113,7 +113,7 @@ module.exports = function (config) {
             [require('enb-bh/techs/bemjson-to-html')]<% } %><% if (technologies.inTechs['tidy.html']) { %>,
             // tidy html
             [require('enb-beautify/techs/enb-beautify-html'), {
-                sourceTarget: '?.html',
+                htmlFileTarget: '?.html',
```
Comment (blond) [inline app/templates/.enb/make.js]: `htmlFile` is better

File: app/templates/.enb/make.js [hunk 2]
```diff
@@ -113,7 +113,7 @@ module.exports = function (config) {
...
                 destTarget: '?.tidy.html'
```
Comment (blond) [inline app/templates/.enb/make.js]: here you can change `destTarget` to `target`

**Comment by blond** [comment]
@Realetive, are you going to fix review comments or we can close this PR?

**Comment by Realetive** [comment]
I think, that you can closed it. Thank you.

**Pull request closed without merging**